specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2025-04-01/computeRPCommon.json

```diff
@@ -0,0 +1,2108 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ComputeManagementClient",
+    "description": "The Compute Management Client.",
+    "version": "2025-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Compute/operations": {
+      "get": {
+        "tags": [
+          "ComputeOperations"
+        ],
+        "operationId": "Operations_List",
+        "description": "Gets a list of compute operations.",
+        "parameters": [
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ComputeOperationListResult"
```

Comment:
> "$ref": "#/definitions/ComputeOperationListResult" [](http://example.com/codeflow?start=13&length=51)

shouldn't it be common-types/resource-management/v5 or v6 /types.json#/definitions/OperationListResult